custom_components/google_photos/camera.py

```diff
@@ -128,6 +129,22 @@ def _handle_coordinator_update(self) -> None:
             self._attr_extra_state_attributes["media_url"] = (
                 media.get("productUrl") or {}
             )
+            if media_secondary is not None:
+                self._attr_extra_state_attributes["media_secondary"] = {"media_filename":(
+                    media_secondary.media.get("filename") or ""
+                ),
+                "media_metadata":(
+                    media_secondary.media.get("mediaMetadata") or {}
+                ),
+                "media_contributor_info": (
+                    media_secondary.media.get("contributorInfo") or {}
+                ),
+                "productUrl":(
+                    media_secondary.media.get("productUrl") or {}
+                )}
+            else:
+                self._attr_extra_state_attributes["media_secondary"] = ''
```

Comment:
Instead of setting an empty string, just remove the key from the dictionary:
```
self._attr_extra_state_attributes.pop("media_secondary", None)
```